arrows/klv/klv_0104.cxx

```diff
@@ -183,9 +183,8 @@ klv_0104::klv_0104()
   m_traitsvec[SLANT_RANGE] =                 NEW_TRAIT( double,       "Slant range" );
   m_traitsvec[ANGLE_TO_NORTH] =              NEW_TRAIT( double,       "Angle to north" );
   m_traitsvec[OBLIQUITY_ANGLE] =             NEW_TRAIT( double,       "Obliquity angle" );
-  m_traitsvec[START_DATE_TIME_UTC] =         NEW_TRAIT( std::string,  "Start Date Time - UTC" );
-  m_traitsvec[EVENT_START_DATE_TIME_UTC] =   NEW_TRAIT( std::string,  "Event Start Date Time - UTC" );
-  m_traitsvec[MISSION_START_TIME] =          NEW_TRAIT( std::string,  "Mission Start Date Time - UTC" );
+  m_traitsvec[START_DATETIME] =              NEW_TRAIT( std::string,  "Start Datetime" );
+  m_traitsvec[EVENT_START_DATETIME] =        NEW_TRAIT( std::string,  "Event Start Datetime" );
```

Comment:
Why is this a string type rather than a uint64?

Reply:
Good question - here it needs to be `string` because it needs to be read in as a string (that's the format it is in the KLV data). In `convert_0104_metadata.cxx` around line 220, you can see the string is taken and converted to a `uint64_t` and stored that way in the final KWIVER metadata, so this is just a temporary state.

Reply:
Great, thanks